frontend/scss/_variables.scss

```diff
@@ -20,6 +20,9 @@ $sizes: (
   xl: '5',
 );
 
+$mobileHeaderHeight: 55px;
```

Comment:
Please add a comment where and why those variables are used.

Reply:
Please check if the suggested comment will do it, otherwise let me know.